integration_tests/test_upgrade.py

```diff
@@ -318,6 +318,34 @@ def test_manual_upgrade_all(cosmovisor_cluster):
 
     assert rsp["params"]["minTimeoutDuration"] == "3600s", rsp
 
+    rsp = json.loads(
+        cli.raw(
+            "query",
+            "params",
+            "subspace",
+            "wasm",
+            "uploadAccess",
+            home=cli.data_dir,
+            node=cli.node_rpc,
+            output="json",
+        )
+    )
+    assert rsp["value"] == '{"permission":"Nobody"}', rsp
```

Comment:
could also try to upload in the upgrade and see if it fails + make a gov proposal to change it later and check it works?